include/votca/tools/histogramnew.h

```diff
@@ -144,7 +137,7 @@ class HistogramNew {
    * \brief get the grid of histogram
    * \return step per bin
    */
-  double getStep() const { return _periodic ? _step_p : _step; }
+  double getStep() const { return _step; }
```

Comment:
Does this not cause problems with the unit tests? I would also suggest you check the csg tests as well as some of them might depend on this functionality.

Reply:
I ran all tests locally